src/basic/sysctl-util.c

```diff
@@ -58,10 +58,13 @@ int sysctl_write(const char *property, const char *value) {
 
         log_debug("Setting '%s' to '%s'", p, value);
 
+        if (cache)
+                hashmap_put(cache, strdup(p), (void *)strdup(value));
```

Comment:
oom checks...

Reply:
also really should use hashmap_ensure_put()

Reply:
the `ops` argument should be `&string_hash_ops_free_free` even when inserting?

Reply:
`hashmap_ensure_put` allocates the hashmap if it's NULL?
Why is this necessary if we do it in `manager_new()`?